tests/plugins/test_add_filesystem.py

```diff
@@ -407,12 +445,11 @@ def test_image_build_overwrites(tmpdir):
                     [fedora-os]
                     baseurl = http://default-repo.com/fedora/$basearch/os.repo
                     """))
-    plugin = create_plugin_instance(tmpdir, {'repos': repos})
+    plugin = create_plugin_instance(tmpdir, {'repos': repos, 'architectures': architectures})
     image_build_conf = dedent("""\
         [image-build]
         name = my-name
         version = 1.0
-        arches = i386,i486
         target = guest-fedora-23-docker
         install_tree = http://install-tree.com/$arch/fedora23/
         format = locker,mocker
```

Comment:
So I didn't add additional cases, per se, but rather, took the existing ones, and inserted architecture into some of them. Seems to work!